addon/components/nypr-accounts/basic-card.js

```diff
@@ -182,9 +182,11 @@ export default Component.extend({
   }).drop(),
 
   promptForPassword: task(function * () {
+    Ember.$('body').addClass('has-nypr-account-modal-open');
     try {
       yield waitForEvent(this, 'passwordVerified');
     } finally {
+      Ember.$('body').removeClass('has-nypr-account-modal-open');
       set(this, 'password', null);
     }
```

Comment:
We don't need a `willDestroyElement` handler for this, because in ember-concurrency tasks, the `finally` block always runs, even if the component is destroyed.  
https://ember-concurrency.com/#/docs/error-vs-cancelation